tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java

```diff
@@ -474,10 +474,12 @@ private static void testMouseWheel(int amount) {
         InvalidationListener invalidationListener = observable -> setSceneLatch.countDown();
         int[] totalScroll = new int[]{0};
         long[] firstScrollMillis = new long[]{0};
+        // The scroll wheel amount is multiplied by 40 on Linux and Windows, but not on macOS. The
+        // directions are also reversed.
```

Comment:
I filed JDK-8214580 to investigate this. Can you add to this comment that the difference in behavior is unexpected and might be a bug? Something like:

```
// This difference in behavior is unexplained and might be a bug. See JDK-8214580.
```